Applications/Analyze/test/subject01_adjusted_spring.osim

```diff
@@ -1,1611 +1,1444 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<OpenSimDocument Version="30503">
+<OpenSimDocument Version="30000">
```

Comment:
I wish that were the case. Here is part of the commit message for the commit in which I reverted the model:

> This commit reverts the model used in the testStaticOptimization passive force
test to version 30000. Previously, the model was at version 30503, but we
changed the Station connector's name from reference_frame to parent_frame after
30503 was first created.

This model file uses the outdated `reference_frame` as the name for Station's Connector. Unfortunately, there is no updateFromXML code that converts `reference_frame` to `parent_frame`. So it is not good to have this intermediate model version in the repo (we should also separately implement `Station::updateFromXMLNode()`).